tests/integration/goldens/redis/samples/generated_samples/redis_generated_redis_v1_cloud_redis_update_instance_async.py

```diff
@@ -34,7 +34,13 @@ async def sample_update_instance():
     client = redis_v1.CloudRedisAsyncClient()
 
     # Initialize request argument(s)
+    instance = redis_v1.Instance()
+    instance.name = "name_value"
+    instance.tier = "STANDARD_HA"
+    instance.memory_size_gb = 1499
```

Comment:
i'm very curious at why 1499 is this generated value? Is it particularly large and could it get someone into trouble by initializing something very expensive?

Reply:
The number is deterministically generated from the name of the field. It's re-using the same logic that's used for unit tests. The numbers are fine for unit tests since no real requests get sent, but it might not be that great for samples...

https://github.com/googleapis/gapic-generator-python/blob/5c8eeaf8118932d63729a18f387154aec0b1d6c7/gapic/schema/wrappers.py#L115-L118

 🤔  @amanda-tarafa How does C# generate mock numeric values for snippetgen?

Reply:
We use the default values for all number type, which is 0. This has the same drawback as generating something as you are, which is, it may or may not work. I think this is fine for now. For generating more meaningfull values, we need configs to be specified, so we can do this for next iterations.

Reply:
Also, we are adding this note for each snippet:

```
/// This snippet has been automatically generated for illustrative purposes only.
/// It may require modifications to work in your environment.
```

Reply:
OK, I will stick with the somewhat random looking numbers, but if anyone thinks a set value (0, 1) would be better it should be an easy change.